charts/opentelemetry-collector/templates/NOTES.txt

```diff
@@ -59,3 +59,7 @@
 {{- if or (and (.Values.internalTelemetryViaOTLP.logs.enabled) (not (or .Values.internalTelemetryViaOTLP.endpoint .Values.internalTelemetryViaOTLP.logs.endpoint))) }}
 {{ fail "[ERROR] internalTelemetryViaOTLP.endpoint or .Values.internalTelemetryViaOTLP.logs.endpoint must be set when internalTelemetryViaOTLP.logs.enabled is true" }}
 {{ end }}
+
+{{- if .Values.config.service.telemetry.metrics }}
+[WARNING] config.service.telemetry.metrics should not be used and will be removed in the future. Use internalTelemetryViaOTLP instead"
```

Comment:
maybe we should make it clear here that we're rewriting this here and we eventually will not?

Reply:
Maybe ripping of the band aid not supporting the old format at all and throwing a "Compile error" with fail would be simpler and not as complicated in the long run.